src/frontend/components/Chat/SharedChatComponents/ChatConversationDisplayer/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Spinner } from 'grommet';
+import React, { useEffect, useRef, useState } from 'react';
+
+import { ChatMessageItem } from 'components/Chat/SharedChatComponents/ChatMessageItem';
+import { useMessagesState } from 'data/stores/useMessagesStore';
+
+export const ChatConversationDisplayer = () => {
+  const { messages } = useMessagesState();
+  const [loading, setLoading] = useState(true);
+  const scrollableContainerRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false);
+      scrollableContainerRef.current!.lastElementChild?.scrollIntoView();
+    }, 3000);
+  }, []);
+
+  const isScrollInBottom = scrollableContainerRef.current
+    ? scrollableContainerRef.current!.scrollHeight -
+        Math.round(scrollableContainerRef.current!.scrollTop) ===
+      scrollableContainerRef.current!.clientHeight
+    : false;
+
+  useEffect(() => {
+    if (
+      scrollableContainerRef.current &&
+      scrollableContainerRef.current!.lastChild &&
+      isScrollInBottom
+    ) {
+      scrollableContainerRef.current!.lastElementChild?.scrollIntoView();
```

Comment:
The idea was good but using `scrollIntoView` is not what we want. Doing this, If I navigate in the page and I don't have the chat in my visible part, The browser will take the focus on the chat every time I receive a new message.
You should take a look at waht was made in the `TranscriptReader` component. We had the same issue and we used a different solution

Reply:
Yes, i changed it to act as in the `TranscriptReader` component and made it use `scrollTo()` instead of `scrollIntoView`.
in commit [7df55a](https://github.com/openfun/marsha/pull/1247/commits/7df55a6730c4fe3e971001452e51243a48e8eae3)